src/yup-schemas/base-yup-schema-helper.ts

```diff
@@ -0,0 +1,74 @@
+import {
+    EElementType,
+    IFocusedElement,
+    TElementMap,
+} from "src/context-providers";
+import * as yup from "yup";
+
+const ID_REGEX = /^[a-z]+(?:[A-Z0-9][a-z0-9]*)*(?:[-_][a-z0-9]+)*$/gm;
+export const PREFILL_ACTIONID_REGEX = /^[a-zA-Z0-9_-]+$/;
+export const PREFILL_PATH_REGEX = /^[a-zA-Z0-9._-]+$/;
```

Comment:
understand it's not part of the scope of this PR, for your next PR, you can consider extracting all the regexes from various files into a separate file